diff -aruN dnf-4.7.0/dnf/const.py.in dnf-4.7.0-new/dnf/const.py.in
--- dnf-4.7.0/dnf/const.py.in	2021-04-12 18:26:33.000000000 -0700
+++ dnf-4.7.0-new/dnf/const.py.in	2022-03-29 10:30:33.000000000 -0700
@@ -55,4 +55,4 @@
 USER_AGENT = "dnf/%s" % VERSION
 
 BUGTRACKER_COMPONENT=NAME.lower()
-BUGTRACKER='https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=%s' % BUGTRACKER_COMPONENT
+BUGTRACKER='https://bugs.rockylinux.org/'
diff -aruN dnf-4.7.0/doc/conf.py.in dnf-4.7.0-new/doc/conf.py.in
--- dnf-4.7.0/doc/conf.py.in	2021-04-12 18:26:33.000000000 -0700
+++ dnf-4.7.0-new/doc/conf.py.in	2022-03-29 10:34:07.000000000 -0700
@@ -267,5 +267,5 @@
 .. _DNF: https://github.com/rpm-software-management/dnf/
 .. _hawkey: http://rpm-software-management.github.io/hawkey/
 .. _YUM: http://yum.baseurl.org/
-.. _bugzilla: https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=dnf
+.. _bugzilla: https://bugs.rockylinux.org/
 """
diff -aruN dnf-4.7.0/tests/test_config.py dnf-4.7.0-new/tests/test_config.py
--- dnf-4.7.0/tests/test_config.py	2021-04-12 18:26:33.000000000 -0700
+++ dnf-4.7.0-new/tests/test_config.py	2022-03-29 10:33:24.000000000 -0700
@@ -55,8 +55,7 @@
     def test_bugtracker(self):
         conf = Conf()
         self.assertEqual(conf.bugtracker_url,
-                         "https://bugzilla.redhat.com/enter_bug.cgi" +
-                         "?product=Fedora&component=dnf")
+                         "https://bugs.rockylinux.org/")
 
     def test_conf_from_file(self):
         conf = Conf()
